scripts/build.sh

```diff
@@ -20,6 +19,7 @@ else
 fi
 export FM_TMP_DIR
 export FM_TEST_FAST_WEAK_CRYPTO="1"
+export FM_POLL_INTERVAL=1
```

Comment:
Define this in one place instead of a few places. For tmuxinator it was set to `0.5` and elsewhere it was `1`. Is setting to `1` ok?